frontend/src/app/components/containers/complaints/complaint-filter.tsx

```diff
@@ -49,6 +49,7 @@ export const ComplaintFilter: FC<Props> = ({ type }) => {
     officers.unshift({ value: "Unassigned", label: "Unassigned" });
   }
   const agency = getUserAgency();
+  const officersByAgency = useAppSelector(selectOfficersByAgencyDropdown(agency));
```

Comment:
Almost... You dropped some of the logic off the officers object that puts Unassigned at the top of the list (this is why Cypress tests are important)

You probably could just modify line 47 to use this selector and then you don't need this line, or the change you made below.

Reply:
For some reason I'd thought this officers listing was used elsewhere so needed a separate instance. I've since renamed it as officersByAgency to indicate what it holds and combined the 'Unassigned' code with that.